common/pulp/common/error_codes.py

```diff
@@ -113,6 +113,9 @@
 PLP0043 = Error("PLP0043", _("Database 'write_concern' config can only be 'majority' or 'all'. "
                              "Refer to /etc/pulp/server.conf for proper use."), [])
 PLP0044 = Error("PLP0033", _("The target importer does not support the types from the source"), [])
+PLP0045 = Error("PLP0045", _("The importer %(importer_id)s indicated a failed response when "
+                             "uploading %(unit_type)s unit to repository %(repository_id)s."),
+                ['importer_id', 'unit_type', 'repository_id'])
```

Comment:
Well, I guess nothing says it's wrong to have a skip in the number schemes, but since the user-facing number for 0044 is actually 0033, it might make sense to fix that here.